controller.py

```diff
@@ -0,0 +1,24 @@
+import interfeace
+import model
+
+
+def run():
+    num = interfeace.menu()
+    if num == '1':
+        data = (model.show_cont())
+        interfeace.result(data,'')
+    elif num == '2':
+        data = interfeace.add_cont()
+        interfeace.result('Добавили запись', model.add_cont(data))
+        # print('Добавили запись', model.add_cont(data))
+    elif num == '3':
+        find_ = interfeace.find_cont()
+        data = model.find_cont(find_)
+        if data == '-1':
+            data = 'Контакт не найден'
+        interfeace.result('Искомый контакт: ', data)
+    elif num == '4':
+        find_ = interfeace.find_cont()
+        model.del_cont(find_)
+    elif num == '5':
+        exit()
```

Comment:
Я несколько раз говорил, что таким образом останавливать работу программы не стоит. Лучше так:
```suggestion
        sys.exit()
```

Reply:
Работа вашей программы не зациклена, то есть один запуск, одно действие.